sql-ballerina/client.bal

```diff
@@ -22,7 +22,7 @@ public type Client client object {
 
     # Queries the database with the query provided by the user, and returns the result as stream.
     #
-    # + sqlQuery - The query which needs to be executed as `string` or `ParameterizedQuery` when the SQL query has
+    # + sqlQuery - The query which needs to be executed as `string` or `sql:ParameterizedQuery` when the SQL query has
```

Comment:
```suggestion
    # + sqlQuery - The query, which needs to be executed as a `string` or an `sql:ParameterizedQuery` when the SQL query has
```

Reply:
Change Added in https://github.com/ballerina-platform/module-ballerina-sql/pull/228/commits/2656f12c62276e15a256c133465cb60304d0b041